src/components/MapPage.js

```diff
@@ -46,14 +46,16 @@ export class MapPage extends Component {
   render() {
     return (
       <Div100vh>
-      <div className="map-page">
-        <Head legendButton={this.legendButton} />
-        <ReactGoogleMaps
-          tapsDisplayed={this.state.tapsDisplayed}
-          ada={this.state.ada}
-          filtered={this.state.filtered}
-        />
-      </div>
+        <div className={styles.mapPage}>
+          <Head legendButton={this.legendButton} />
+          <div className={styles.mapWrapper}>
+            <ReactGoogleMaps
+              tapsDisplayed={this.state.tapsDisplayed}
+              ada={this.state.ada}
+              filtered={this.state.filtered}
+            />
+          </div>
```

Comment:
The map was getting pushed down off the page and clipping the zoom controls, so this new wrapper div allows the map to be properly sized and positioned below the header. An extra wrapper div gets added automatically around the map without any class name, making it inconvenient to style, hence the need for this additional wrapper div with a class applied to it. There are issues opened in the google-maps-react project referencing this issue and there seems to be no good resolution for it.

Reply:
Since the app header was moved off into the App component in develop, I made a small change since I was getting a double-header in the test site.